### ISSUE TITLE
Figma's ebuild fails the checksum verification

### ISSUE THREAD

**Issue by mbaraa** [night-every/nightfreedom-overlay#1]
When I'm trying to install figma, using the provided ebuild, it show the following error

```
Would you like to merge these packages? [Yes/No] 
>>> Verifying ebuild manifests
>>> Emerging (1 of 1) media-gfx/figma-linux-bin-0.11.2::nightfreedom-overlay
>>> Failed to emerge media-gfx/figma-linux-bin-0.11.2, Log file:
>>>  '/var/tmp/portage/media-gfx/figma-linux-bin-0.11.2/temp/build.log'
>>> Jobs: 0 of 1 complete, 1 failed                 Load avg: 3.40, 4.00, 4.85
!!! Fetched file: figma-linux-bin-0.11.2-amd64.deb VERIFY FAILED!
!!! Reason: Insufficient data for checksum verification
!!! Got:      
!!! Expected: BLAKE2B BLAKE2S MD5 RMD160 SHA1 SHA256 SHA3_256 SHA3_512 SHA512 WHIRLPOOL
```

I tried resyncing the repo, updating portage, but nothing worked!
So what seems to be the problem?

**Comment by mbaraa** [comment]
@night-every 
I'm guessing it got fixed in the commit [2238324](https://github.com/night-every/nightfreedom-overlay/commit/2238324d5bbe9c5782e4f39718ec34016a0bc410)

**Comment by mbaraa** [comment]
UPDATE: it was fixed, thanks!
